### PR TITLE
[plugin-next] Fix Rollup when no template in an SFC

### PR DESCRIPTION
**Pull request by elevatebart**
Changes proposed in this pull request:
SFCs can have a script block only
In this case, the render function should not be replaced by the empty template function.
It should be left alone.

/ping @znck @jtommy

### CODE REVIEW

**Comment by jtommy** [comment]
@elevatebart Yes, removing it seems to work as I tried manually but I don't know if it's really correct or not in the new Vue 3.

File: src/index.ts [hunk 1]
```diff
@@ -417,13 +417,17 @@ function transformVueSFC(
...
   const hasScoped = descriptor.styles.some((s) => s.scoped)
-  const templateImport = getTemplateCode(
-    descriptor,
-    resourcePath,
-    id,
-    hasScoped,
-    isServer
-  )
+
+  const templateImport = descriptor.template
```
Comment (jtommy) [inline src/index.ts:421]: The right condition is `!descriptor.template`

File: examples/no-template/src/HelloWorld.vue [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<script>
+import h from 'vue'
```
Comment (jtommy) [inline examples/no-template/src/HelloWorld.vue:2]: h isnt' the default export, you should use `import { h } from 'vue'`

**Review by jtommy — CHANGES_REQUESTED** [review]

**Review by znck — APPROVED** [review]

**Comment by meteorlxy** [comment]
Some edge cases that may be related to https://github.com/vuejs/vue-next/issues/2463

**Comment by zouhangwithsweet** [comment]
I am troubled by this question.
When will you release the new version, maybe beta.11 ?:clap:

**Pull request merged**